CPlusPlus/Sample_Source/DocumentConversion/ConvertToFactur-X/ConvertToFactur-X.cpp

```diff
@@ -0,0 +1,330 @@
+//
```

Comment:
The FacturXConverter sample also needs to be added to the Java/Sample_Source/DocumentConversion directory.